_layouts/default.html

```diff
@@ -84,7 +84,7 @@
         </a>
         </div>
         <div class="column">
-          <a href="https://www.katacoda.com/move2kube" class="icon-link">
+          <a href="https://move2kube.konveyor.io" class="icon-link">
             <img alt="Move2Kube on Katacoda" src="{{ site.baseurl }}/assets/images/katacoda-icon.png" width="60" height="60" title="Katacoda">
           </a>
         </div>
```

Comment:
We should remove the Katacoda icon from Move2Kube's website as Katacoda.com is closed.